### PR TITLE
Fix comments; Clarify in comment that _item_key and resource both contain the hold id in FOLIO

### PR DESCRIPTION
**Pull request by corylown**

### CODE REVIEW

File: app/services/folio_client.rb [hunk 1]
```diff
@@ -119,8 +119,8 @@ def renew_item_request(user_id, item_id)
   end
 
   # API compatibility shim with SymphonyClient
-  # @param [String] resource the UUID of the hold in FOLIO
-  # @param [String] _item_key the UUID of the FOLIO item; this was required by Symphony.
+  # @param [String] resource the UUID of the FOLIO hold
+  # @param [String] _item_key the UUID of the FOLIO hold; this was the item key in Symphony
```
Comment (jcoyne) [inline app/services/folio_client.rb:123]:
```suggestion
  # @param [String] _item_key this was the item key in Symphony
```

File: app/services/folio_client.rb [hunk 2]
```diff
@@ -161,17 +161,17 @@ def change_pickup_library(hold_id, service_point)
   # TODO: after updating feature tests for FOLIO remove this shim and use #change_pickup_expiration directly
   #
   # API compatibility shim with SymphonyClient
-  # @param [String] resource the UUID of the hold in FOLIO
-  # @param [String] _item_key the UUID of the FOLIO item; this was required by Symphony.
-  # @param [String] patron_key the UUID of the user in FOLIO
+  # @param [String] resource the UUID of the FOLIO hold
+  # @param [String] _item_key the UUID of the FOLIO hold; this was the item key in Symphony
```
Comment (jcoyne) [inline app/services/folio_client.rb:165]:
```suggestion
  # @param [String] _item_key this was the item key in Symphony
```

**Review by jcoyne — CHANGES_REQUESTED** [review]
Small suggestion to improve the comments.

**Comment by corylown** [comment]
> Small suggestion to improve the comments.

I'm fine with the proposed change if we merge this PR to consistently use the resource param when we need the hold UUID in FOLIO: https://github.com/sul-dlss/mylibrary/pull/959

**Pull request merged**